File: ApiSuvesaPos/Negocio/Modelo/caja/cierracaja.vb
Namespace Modelo
    Public Class cierrecaja
        Public Property NumeroCierre As Integer
        Public Property Cajera As String
        Public Property Nombre As String
        Public Property Apertura As Integer
        Public Property Usuario As String
        Public Property Fecha As DateTime
        Public Property NombreUsuario As String
        Public Property Anulado As Boolean
        Public Property Devoluciones As Decimal
        Public Property Subtotal As Decimal
        Public Property TotalSistema As Decimal
        Public Property Equivalencia As Decimal
        Public Property EfectivoColones As Decimal
        Public Property EfectivoDolares As Decimal
        Public Property TarjetaColones As Decimal
        Public Property TarjetaDolares As Decimal
        Public Property Cheques As Decimal
        Public Property ChequeDol As Decimal
        Public Property DepositosCol As Decimal
        Public Property DepositosDol As Decimal
        Public Property CierreCaja_DetMon As New List(Of Modelo.cierrecaja_detmon)
        Public Property CierreCaja_DetTarj As New List(Of Modelo.cierrecaja_dettarj)
    End Class

    Public Class cierrecaja_detmon
        Public Property Id_DetaMoneda As Integer
        Public Property Id_CierreCaja As Integer
        Public Property Id_Moneda As Integer
        Public Property MontoSistema As Decimal
        Public Property MontoCajero As Decimal
    End Class

    Public Class cierrecaja_dettarj
        Public Property Id_DetalleTarjeta As Integer
        Public Property Id_CierreCaja As Integer
        Public Property Id_TipoTarjeta As Integer
        Public Property MontoSistema As Decimal
        Public Property MontoCajero As Decimal
    End Class

End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/caja/cierracaja.cs
using System;
using System.Collections.Generic;

namespace Modelo
{
    public class cierrecaja
    {
        public int NumeroCierre { get; set; }
        public string Cajera { get; set; }
        public string Nombre { get; set; }
        public int Apertura { get; set; }
        public string Usuario { get; set; }
        public DateTime Fecha { get; set; }
        public string NombreUsuario { get; set; }
        public bool Anulado { get; set; }
        public decimal Devoluciones { get; set; }
        public decimal Subtotal { get; set; }
        public decimal TotalSistema { get; set; }
        public decimal Equivalencia { get; set; }
        public decimal EfectivoColones { get; set; }
        public decimal EfectivoDolares { get; set; }
        public decimal TarjetaColones { get; set; }
        public decimal TarjetaDolares { get; set; }
        public decimal Cheques { get; set; }
        public decimal ChequeDol { get; set; }
        public decimal DepositosCol { get; set; }
        public decimal DepositosDol { get; set; }
        public List<Modelo.cierrecaja_detmon> CierreCaja_DetMon { get; set; } = new List<Modelo.cierrecaja_detmon>();
        public List<Modelo.cierrecaja_dettarj> CierreCaja_DetTarj { get; set; } = new List<Modelo.cierrecaja_dettarj>();
    }

    public class cierrecaja_detmon
    {
        public int Id_DetaMoneda { get; set; }
        public int Id_CierreCaja { get; set; }
        public int Id_Moneda { get; set; }
        public decimal MontoSistema { get; set; }
        public decimal MontoCajero { get; set; }
    }

    public class cierrecaja_dettarj
    {
        public int Id_DetalleTarjeta { get; set; }
        public int Id_CierreCaja { get; set; }
        public int Id_TipoTarjeta { get; set; }
        public decimal MontoSistema { get; set; }
        public decimal MontoCajero { get; set; }
    }
}
